2020-03-18 13:18:09,858 - instabot version: 0.115.0 (bot) - INFO - Instabot version: 0.115.0 Started
2020-03-18 13:18:09,858 - instabot version: 0.115.0 (bot) - DEBUG - Bot imported from /usr/local/lib/python2.7/dist-packages/instabot/bot/bot.pyc
2020-03-18 13:18:09,860 - instabot version: 0.115.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2020-03-18 13:18:10,491 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2020-03-18 13:18:11,293 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-03-18 13:18:11,814 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-03-18 13:18:12,387 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2020-03-18 13:18:14,762 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2020-03-18 13:18:14,763 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "The password you entered is incorrect. Please try again.", "invalid_credentials": true, "error_title": "Incorrect password for nurchuliss", "buttons": [{"title": "Try Again", "action": "dismiss"}], "status": "fail", "error_type": "bad_password"}
2020-03-18 13:18:14,763 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-18 13:18:14,763 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: The password you entered is incorrect. Please try again.
2020-03-18 13:18:14,764 - instabot version: 0.115.0 (api) - INFO - Error type: bad_password
2020-03-18 13:18:14,764 - instabot version: 0.115.0 (api) - ERROR - Failed to login go to instagram and change your password
2020-03-18 13:18:14,764 - instabot version: 0.115.0 (api) - INFO - Username or password is incorrect.
